src/common/memContext.c

```diff
@@ -40,13 +40,21 @@ typedef struct MemContextAlloc
 Contains information about the memory context
 ***********************************************************************************************************************************/
 // Quantity of child contexts, allocations, or callbacks
+#ifndef _WIN32
```

Comment:
Is it possible to test for MSVC rather than win32?  It would keep the options open for using the gnu compiler on windows.